apps/sap-commerce-cloud/vite.config.ts

```diff
@@ -0,0 +1,16 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+
+export default defineConfig(() => ({
```

Comment:
I was checking the MS teams vite config to base most of this off of. We didn't have to do anything special with the build since this is purely a frontend app. I'm particularly unsure if the `test` config setup is a correct/generic setup or if that is something special to the MS teams app and we want to do something different here.